src/model_backend/model_api.cpp

```diff
@@ -179,3 +180,37 @@ std::string model_api::generation_with_image(const std::string &prompt, const st
     ollama::response response = ollama::generate(model, prompt, options, image);
     return response.as_simple_string(); // Return the response as a string
 }
+
+std::string ModelApi::TranscriptionService(std::vector<std::string> args)
+{
+    // pass args for transciption service in format ({"parameter_name","param_value","param_name","param_value"})
+    // check transcription_service.hpp to see all the parameters that can be set, for more info look at whisper.cpp repo
+
+    std::string whisper_cpp_path = "../../src/model_backend/thirdparty/whisper.cpp";
```

Comment:
app will be installed at /usr/bin. Please change this so that not at this location, cause this path will not work